mypy.ini

```diff
@@ -66,6 +66,8 @@ no_implicit_reexport=True
 follow_imports = skip
 
 
+[mypy-celery.*]
```

Comment:
What does this and the line below it do?

Reply:
mypy does not really understand dependencies for one reason or another (either the dependency not being installed while mypy runs or mypy being dumb)

For every dependency we use in a typed module we need to tell mypy to stop worrying as to whether the thing we're importing actually exists